revolv/base/forms.py

```diff
@@ -21,6 +21,7 @@ def save(self, commit=True):
         last names.
         """
         user = super(SignupForm, self).save(commit=False)
+        user.save()
```

Comment:
oops my bad there was actually an error somewhere else so i just reverted this back to how it was before